manifests/v/VSRevoGroup/RevoUninstallerPro/5.0.1.0/VSRevoGroup.RevoUninstallerPro.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: VSRevoGroup.RevoUninstallerPro
+PackageVersion: 5.0.1.0
+PackageLocale: en-US
+Publisher: VS Revo Group

```

```suggestion
Publisher: VS Revo Group, Ltd.
```
Should match control panel